File: chapitre-01/exemple-argv.c
// ------------------------------------------------------------------
// exemple-argv.c
// Fichier d'exemple du livre "Developpement Systeme sous Linux"
// (C) 2000-2019 - Christophe BLAESS <christophe@blaess.fr>
// https://www.blaess.fr/christophe/
// ------------------------------------------------------------------

#include <stdio.h>

int main (int argc, char * argv[])
{
	int	i;

	fprintf(stdout, "%s a recu en argument :\n", argv[0]);
	for (i = 1; i < argc; i ++)
		fprintf(stdout, "  %s\n", argv[i]);
	return 0;
}

File: chapitre-04/exemple-argv.py
#! /usr/bin/python
# ------------------------------------------------------------------
# exemple-argv.py
# Fichier d'exemple du livre "Developpement Systeme sous Linux"
# (C) 2015-2019 - Christophe BLAESS <christophe@blaess.fr>
# https://www.blaess.fr/christophe/
# ------------------------------------------------------------------

from __future__ import print_function
import sys

print(sys.argv[0], "a recu en argument :")
sys.argv.pop(0)
for var in sys.argv:
	print(" ", var)
